File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
 
 {% endtabs %} 
 
+### Link Paragraph and Character Style
+
+You can link paragraph and character style using the `LinkedStyleName` property.
+
+The following code example explains how to link paragraph and character style.
+
+{% tabs %} 
+
+{% highlight c# %}
+//Opens an input Word template
+WordDocument document = new WordDocument();
+//This method adds a section and a paragraph in the document
+document.EnsureMinimal();
+//Adds a new paragraph style named "MyParaStyle"
+IWParagraphStyle myParaStyle = document.AddParagraphStyle("MyParaStyle");
```

Comment:
Change the instance name as "paraStyle"
Change the Style name as "ParagraphStyle"

Reply:
changed